sky/task.py

```diff
@@ -255,6 +255,7 @@ def __init__(
         self.event_callback = event_callback
         # Ignore type error due to a mypy bug.
         # https://github.com/python/mypy/issues/3004
+        self._num_nodes = 1
```

Comment:
Why needed for this upgrade?

Reply:
It is for the mypy issue. Removed the staled comments : )